### DISCUSSION TITLE
Remove the query parameter for retired locales

### DISCUSSION THREAD

**Discussion by franktopel** [mdn/mdn-community#235] [Localization] [answered]
Referring to https://github.com/orgs/mdn/discussions/147

Whenever I as a user of a German-language browser access any MDN page I get this annoying query parameter in the URL: `?retiredLocale=de`

Since I regularly check with MDN docs and c&p links to relevant pages, this keeps me having to remove this from the URL manually.

Is there a technical necessity to have this query parameter in place? I'm kindly asking to remove that.

**Comment by pransh15** [comment, marked as answer]
Closing this as outdated. You can now choose DE as your locale and surf in `https://developer.mozilla.org/de/` as your URL. Feel free to start a new discussion if there's an issue.
